.prettierignore

```diff
@@ -12,4 +12,7 @@ coverage
 dist
 public
 
-*.d.ts
\ No newline at end of file
+*.d.ts
+
+# handlebars templates
+*.hbs
```

Comment:
This works for now, but I wonder if there is anything we can do in the future to make sure that the resulting templates that are generated from these handelbars files are formatted correctly?

Reply:
If it's not somewhere in the tickets it should be - ideally after a template is generated, `lint` and `prettier` will be run so that it is properly formatted. This opens up some discussion about what happens if the user doesn't want those things - I guess we can lint/format with `npx`.